docs/Developer Reference/link-token-contracts.md

```diff
@@ -174,3 +174,20 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
 |Name|ChainLink Token on xDai|
 |Symbol|LINK|
 |Decimals|18|
+
+## Avalanche
+
+### Fuji Testnet
+
+> 🚰Avax Faucet
+>
+> Free Avax for transactions: https://linkfaucet.protofire.io/fuji
```

Comment:
Add fuji community faucet: https://linkfaucet.protofire.io/fuji

Reply:
> Looks just about all set.
> 
> I'm wondering if we can start adding oracles in here too.
> https://market.link/nodes/0cdf5cd6-b296-4230-a4dc-36f9b8323e28/metrics?network=1

I'm okay with that.

Reply:
Do you mean the link is a community faucet or you forgot to add another link?